common/templates/general.go

```diff
@@ -240,6 +241,29 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 				ContentType: "text/plain",
 				Reader:      &buf,
 			}
+
+		case "image":
+			if val == nil {
+				return nil, errors.New("need an argument for the image")
+			}
+			str := fmt.Sprint(val)
+			img, err := http.Get(str)
+			if err != nil {
+				return nil, errors.New("error while trying to get the image")
+			}
+
+			imageName := "image.png"
+			imageType := "image/png"
+			if strings.HasSuffix(strings.ToLower(str), ".gif") {
+				imageName = "image.gif"
+				imageType = "image/gif"
+			}
+			msg.File = &discordgo.File{
+				Name: imageName,
+				ContentType: imageType,
+				Reader: img.Body,
```

Comment:
Have you checked what happens if the attachment is not an image? 
Perhaps this should be handled. 
Dunno if this can be maliciously used.

Reply:
It literally creates a file named image.png without any content. Can't be used maliciously. Tried youtube links and direct links to compressed files that are often used as a virus.
![image](https://user-images.githubusercontent.com/75037449/132967386-b7341b6d-ec75-4448-8eff-6b10e65ab51c.png)

Reply:
I don't think this is ideal, even if it can't be used maliciously. 
Bot should definitely respond with and error, in my opinion.

Reply:
Alright, I'll handle that. Thanks!